example-projects/warp-example/src/main.rs

```diff
@@ -0,0 +1,14 @@
+#![deny(warnings)]
+use cloudevents_sdk_warp::{filter, reply};
+use warp::Filter;
+
+#[tokio::main]
+async fn main() {
+    let routes = warp::any()
+        // extracting event from request
+        .and(filter::to_event())
+        // returning event back
+        .map(|event| reply::from_event(event));
+
+    warp::serve(routes).run(([127, 0, 0, 1], 3030)).await;
```

Comment:
This is not returning any error?

Reply:
I find `warp` error handling a bit odd and probably there is a better way of handling errors in `reply::from_event`. At the moment it mimics `warp::reply::json` which will, in case of serialisation errors, return `HTTP 500`